packages/core/types/src/workflow/fulfillment/create-fulfillment.ts

```diff
@@ -180,4 +180,7 @@ export type CreateFulfillmentWorkflowInput = {
    * The associated fulfillment order.
    */
   order: CreateFulfillmentOrderWorkflowDTO
+
+  // TODO: revisit
```

Comment:
question: Revisit to do what? 😄

Reply:
To either remove `order_id` or `order` (I wasn't sure what was the intention with passing the entire object here - should passing the id to the workflow be sufficient and then we can pass order data to fulfillment service from the workflow?)

Reply:
Cool, I see. I am not entirely sure what the idea is with that, but think it will be more clear once we get to use it.